### PR TITLE
Fix na blacklist

### PR DESCRIPTION
**Pull request by robertzk**

### CODE REVIEW

File: tests/testthat/helper-batch_data.R [hunk 1]
```diff
@@ -14,6 +14,12 @@ batch_data <- function(key, model_version = "model_test", type = "record_id",
   if (add_column) {
     ret$new_col <- 1
   }
+  if (isTRUE(na)) {
+    ret$y <- NA_character_
+  }
+  if (identical(na, "all")) {
+    ret[, colnames(ret) != "id"] <- NA_character_
+  }
```
Comment (peterhurford) [inline tests/testthat/helper-batch_data.R:22]: Is https://github.com/robertzk/cachemeifyoucan/blob/407073846eeb304edeee3eb5257c8c37f960c267/tests/testthat/helper-expect_cached.R#L9-L11 helpful here?

**Comment by peterhurford** [comment]
LGTM :+1: :shipit:

File: R/db.R [hunk 2]
```diff
@@ -284,7 +284,7 @@ write_data_safely <- function(dbconn, tblname, df, key, safe_columns, blacklist)
     if (length(blacklist) > 0L) {
       df_without_id <- df[-which(names(df) == key)]  # Don't blacklist on id.
       if (any(is.na(blacklist))) {
-        df <- df[apply(!is.na(df_without_id), 1, all), , drop = FALSE]
+        df <- df[!apply(is.na(df_without_id), 1, all), , drop = FALSE]
```
Comment (peterhurford) [inline R/db.R:287]: 😨

File: tests/testthat/helper-batch_data.R (hunk 1 above)
Comment (robertzk) [inline tests/testthat/helper-batch_data.R:22]: Should be another way to do it! I want to make sure I'm testing multiple columns though.

**Pull request merged**